File: stephanie/Problema0156/Problema0156.py
preco = input("Digite o preço do produto: ")
codigo = input("Digite o código de origem do produto: ")

if(int(codigo)==1):
    print("Procedência do produto: Sul")
elif(int(codigo)==2):
    print("Procedência do produto: Norte")
elif(int(codigo)==3):
    print("Procedência do produto: Leste")
elif(int(codigo)==4):
    print("Procedência do produto: Oeste")
elif(int(codigo)==5 or int(codigo)==6):
    print("Procedência do produto: Nordeste")
elif(int(codigo)==7 or int(codigo)==8 or int(codigo)==9):
    print("Procedência do produto: Sudeste")
elif(int(codigo)>=10 and int(codigo)<=20):
    print("Procedência do produto: Centro-Oeste")
elif(int(codigo)>=21 and int(codigo)<=30):
    print("Procedência do produto: Nordeste")

File: stephanie/Problema0156/Problema0156.java
package stephanie.Problema0156;

import java.util.Scanner;

public class Problema0156 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.print("Digite o preço do produto: ");
        double preco = Double.parseDouble(input.nextLine());
        System.out.print("Digite o código de origem do produto: ");
        int codigo = Integer.parseInt(input.nextLine());

        if(codigo==1){
           System.out.print("Procedência do produto: Sul");
        }else if(codigo==2){
            System.out.print("Procedência do produto: Norte");
        }else if(codigo==3){
            System.out.print("Procedência do produto: Leste");
        }else if(codigo==4){
            System.out.print("Procedência do produto: Oeste");
        }else if(codigo==5||codigo==6){
            System.out.print("Procedência do produto: Nordeste");
        }else if(codigo==7||codigo==8||codigo==9){
            System.out.print("Procedência do produto: Sudeste");
        }else if(codigo<=10&&codigo<=20){
            System.out.print("Procedência do produto: Centro-Oeste");
        }else if(codigo<=21&&codigo>=30){
            System.out.print("Procedência do produto: Nordeste");
        }
    }
}
